UVA-Relay/sql/Sqlite.cs

```diff
@@ -0,0 +1,194 @@
+                        //canvasIdParameter.ParameterName = "@canvasIdQuery";
+                        cmd.Parameters.Add(discordIdParameter);
+                        cmd.Parameters.Add(guildIdParameter);
+                        //cmd.Parameters.Add(canvasIdParameter);
+                        discordIdParameter.Value = discordId;
+                        guildIdParameter.Value = guildId;
+                        //canvasIdParameter.Value = ;
+                        //Execute query
+                        cmd.ExecuteNonQuery();
+                    }
+                }
+            }
+            catch (SQLiteException exc)
+            {
+                //Console.WriteLine(exc.Message);
+                Console.WriteLine("User already in database or error has occured");
+
+            }
+        }
+        //Update guild settings to database
+        public void UpdateGuildSettings(long guildId, long value, string option)
+        {
+            var s = new SQLiteConnection(@_dbFile);
+            try
+            {
+                using (s)
+                {
+                    using (var cmd = s.CreateCommand())
+                    {
+                        s.Open();
+                        cmd.CommandText = $"UPDATE GUILD SET {option} = @valueQuery WHERE GuildId = @GuildIdQuery";
+                        var optionParam = cmd.CreateParameter();
+                        var guildIdParameter = cmd.CreateParameter();
+                        var valueParameter = cmd.CreateParameter();
+                        optionParam.ParameterName = "@optionQuery";
+                        guildIdParameter.ParameterName = "@GuildIdQuery";
+                        valueParameter.ParameterName = "@valueQuery";
+                        cmd.Parameters.Add(optionParam);
+                        cmd.Parameters.Add(guildIdParameter);
+                        cmd.Parameters.Add(valueParameter);
+                        optionParam.Value = option;
+                        guildIdParameter.Value = guildId;
+                        valueParameter.Value = value;
+                        //Execute query
+                        cmd.ExecuteNonQuery();
+                    }
+                }
+            }
+            catch (SQLiteException exc)
+            {
+                Console.WriteLine(exc.Message);
+                //Console.WriteLine("Error has occurred");
+
+            }
+        }
+        //Update guild settings to database
+        public void UpdateDiscordUserSettings(string discordId, string guildId, long value, string option)
+        {
+            var s = new SQLiteConnection(@_dbFile);
+            try
+            {
+                using (s)
+                {
+                    using (var cmd = s.CreateCommand())
+                    {
+                        s.Open();
+                        cmd.CommandText = $"UPDATE USER SET {option} = @valueQuery WHERE GuildId = @GuildIdQuery AND DiscordID = @DiscordIdQuery";
```

Comment:
The option parameter should be a prepared parameter like the rest. Also, please use the cmd.Parameters.AddWithValue() method for setting prepared parameters.